src/components/export-panel.tsx

```diff
@@ -1,20 +1,20 @@
-import { ReactElement, useRef } from 'react'
+import { ReactElement } from 'react'
 
 import { ExportTypeEnum } from '@core/enum/export-type.enum'
-import { useOutsideClick } from '@core/hooks/use-outside-click'
 import { t } from '@i18n'
 import datasetStore from '@store/dataset'
 import operationsStore from '@store/operations'
 import { showToast } from '@utils/notifications/showToast'
 import { MainTableDataCy } from './data-testid/main-table.cy'
-
-interface Props {
-  close: () => void
-}
-
-export const ExportPanel = ({ close }: Props): ReactElement => {
-  const ref = useRef<any>(null)
-
+import {
+  ISimplePopperModalProps,
+  SimplePopperModal,
+} from './simple-popper-modal/simple-popper-modal'
+import { SimplePopperModalItem } from './simple-popper-modal/simple-popper-modal-item'
+
+export const ExportPanel = ({
```

Comment:
It's modal, may be we can add suffix `Modal` to component name? Like `ExportModal` or `ExportPanelModal`?